docs/zones.md

```diff
@@ -4,39 +4,71 @@ title: Zones
 slug: /zones
 ---
 
-The Platform has 4 data zones, as data moves from the landing to the trusted zone it will become:
+The Platform has 4 data zones, as data is moved from the landing or raw zones to the trusted zone it will become:
```

Comment:
Perhaps we could acknowledge that other organisations might organise their DP zones slightly differently, but at Hackney this is what we have decided to do?